mule-extensions-api/src/main/resources/META-INF/mule-module.properties

```diff
@@ -0,0 +1,30 @@
+module.name=mule-extensions-api
+
+artifact.export.classPackages=org.mule.runtime.extension.api,\
+                              org.mule.runtime.extension.api.annotation,\
+                              org.mule.runtime.extension.api.annotation.capability,\
+                              org.mule.runtime.extension.api.annotation.connector,\
+                              org.mule.runtime.extension.api.annotation.metadata,\
+                              org.mule.runtime.extension.api.annotation.param,\
+                              org.mule.runtime.extension.api.annotation.param.display,\
+                              org.mule.runtime.extension.api.exception,\
+                              org.mule.runtime.extension.api.introspection,\
+                              org.mule.runtime.extension.api.introspection.config,\
+                              org.mule.runtime.extension.api.introspection.connection,\
+                              org.mule.runtime.extension.api.introspection.declaration,\
+                              org.mule.runtime.extension.api.introspection.declaration.fluent,\
+                              org.mule.runtime.extension.api.introspection.declaration.spi,\
+                              org.mule.runtime.extension.api.introspection.declaration.type,\
+                              org.mule.runtime.extension.api.introspection.exception,\
+                              org.mule.runtime.extension.api.introspection.metadata,\
+                              org.mule.runtime.extension.api.introspection.operation,\
+                              org.mule.runtime.extension.api.introspection.parameter,\
+                              org.mule.runtime.extension.api.introspection.property,\
+                              org.mule.runtime.extension.api.introspection.source,\
+                              org.mule.runtime.extension.api.manifest,\
+                              org.mule.runtime.extension.api.resources,\
+                              org.mule.runtime.extension.api.resources.spi,\
+                              org.mule.runtime.extension.api.runtime,\
+                              org.mule.runtime.extension.api.runtime.source,\
+                              org.mule.runtime.extension.internal.manifest
```

Comment:
This is the first iteration on defining the API, exposing everything in the same way we did before. Next iterations will clean up every module to export only the required packages